hack/ocp-install-nightly-art-operators.sh

```diff
@@ -24,11 +24,11 @@ function wait_for_all_nodes_ready() {
   done
 }	
 
-brew_credentials=$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s) #make sure you created a token before (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry)
+brew_credentials=$(curl --cert -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s) #make sure you created a token before (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry)
```

Comment:
Details in slack (https://redhat-internal.slack.com/archives/C04Q3Q8GJ6Q/p1680597167390959), but it doesn't seem like you can just change one into the other without changing additional params